### ISSUE TITLE
Could you please provide code for evaluation on 300W?

### ISSUE THREAD

**Issue by protossw512** [FengZhenhua/Wing-Loss#1]
There are different approaches on normalizing Mean Error, I would like to know how did you achieve this in your evaluation.
The inter-pupil distance is kind of confuse for me, since 68 landmarks do not contain landmarks on pupils, how could we get location of pupil?
Thanks

**Comment by FengZhenhua** [comment]
Hi, for the 300-W dataset, there are two standard protocols. The one I used in the Wing loss paper is from the paper "Face Alignment via Regressing Local Binary Features". If you check this paper, you can find the statement in the paper "For datasets (e.g. 300-W) which have no pupil landmark labeled, we use average position of the landmarks around the eye instead." So you have to use all the landmarks around the eye to calculate pupil positions. 

However, I can provide the code to run the test on the 300-W dataset. I will let you know once I upload the code.

Best,

Zhenhua

**Comment by protossw512** [comment]
@FengZhenhua 
Thank you so much for your reply, it really helped a lot!

**Comment by FengZhenhua** [comment]
Do you still need the code, or you now believe that you can do the evaluation on 300W by yourself correctly? If it is the latter one, I will not spend time on uploading the code :)

**Comment by protossw512** [comment]
@FengZhenhua 
I apologies about previous confusing reply, I believe I can do this by myself, many thanks!

**Comment by FengZhenhua** [comment]
@protossw512 No worries. Then I will just close this issue. Thank you :)

**Comment by epoc88** [comment]
Share the code for Wingloss?
Is this appropriate for 300W, MobileNetV2_0.25 ?

    wing_loss = WingLoss(10.0, 2.0)

    loss = wing_loss(landmarks_batch.to(device), pre_landmarks, euler_angle_weights)

see this 
https://github.com/github-luffy/PFLD_68points_Pytorch/blob/master/train_model.py

the loss did not change at all.

please share your code..
